src/canvas-force-graph.js

```diff
@@ -48,6 +48,7 @@ export default Kapsule({
     linkVisibility: { default: true, triggerUpdate: false },
     linkColor: { default: 'color', triggerUpdate: false },
     linkAutoColorBy: {},
+    linkLineDash: { default: [], triggerUpdate: false },
```

Comment:
Better initialize this as `null` (no default) for the regular non-dashed case.

Reply:
Done